src/storage/src/hummock/event_handler/hummock_event_handler.rs

```diff
@@ -197,7 +197,7 @@ pub struct HummockEventHandler {
     /// A copy of `read_version_mapping` but owned by event handler
     local_read_version_mapping: HashMap<LocalInstanceId, (TableId, HummockReadVersionRef)>,
 
-    version_update_notifier_tx: Arc<tokio::sync::watch::Sender<HummockEpoch>>,
+    version_update_notifier_tx: Arc<tokio::sync::watch::Sender<PinnedVersion>>,
```

Comment:
Holding `PinnVersion` implies that the HummockVersion will be held in memory and also be pinned in meta. IIUC, the item in the watch channel will not be dropped when there is no receiver and no futher update to the channel as long as the sender is not dropped. In this case, the version will be pinned.

Given that we only need the version payload itself for notification, I wonder whether we can just send `Arc<HummockVersion>` instead of `PinnedVersion` here?